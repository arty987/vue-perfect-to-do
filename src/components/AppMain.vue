<script>
import TodoHeader from "@/components/TodoHeader.vue";
import TodoList from "@/components/TodoList.vue";
import TodoFooter from "@/components/TodoFooter.vue";

export default {
  name: "AppMain",
  components: {
    TodoFooter,
    TodoList,
    TodoHeader
  }
}
</script>

<template>
  <main>
    <h1>T O D O</h1>
    <ul>
      <TodoHeader/>
      <TodoList/>
      <TodoFooter/>
    </ul>
  </main>
</template>

<style scoped>

</style>