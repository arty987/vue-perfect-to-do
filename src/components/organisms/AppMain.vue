<script>
import TodoHeader from "@/components/molecules/TodoHeader.vue";
import TodoList from "@/components/molecules/TodoList.vue";
import TodoFooter from "@/components/molecules/TodoFooter.vue";

export default {
  name: "AppMain",
  components: {
    TodoHeader,
    TodoList,
    TodoFooter
  }
}
</script>

<template>
  <main>
    <h1>T O D O</h1>
    <ul>
      <TodoHeader/>
      <TodoList/>
      <TodoFooter/>
    </ul>
  </main>
</template>

<style scoped>

</style>