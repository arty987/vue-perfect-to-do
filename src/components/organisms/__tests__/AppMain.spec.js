import {shallowMount} from '@vue/test-utils'
import AppMain from "@/components/organisms/AppMain.vue";

describe('AppMain.vue', () => {
    it('Should render AppMain', () => {
        const wrapper = shallowMount(AppMain);
        expect(wrapper.element).toMatchSnapshot();
    });
})