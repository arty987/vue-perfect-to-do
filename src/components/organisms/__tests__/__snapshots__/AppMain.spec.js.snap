// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppMain.vue Should render AppMain 1`] = `
<main
  class="app-main"
>
  <todo-header-stub />
  <ul
    class="list-group"
  >
    <todo-input-stub />
    <div
      class="shadow-lg rounded"
    >
      <todo-list-stub />
      <todo-footer-stub />
    </div>
  </ul>
</main>
`;
